Update doc for resource group property of traffic manager endpoint (#7275)
terraform-providers/terraform-provider-azurerm: website/docs/r/traffic_manager_endpoint.html.markdown

> * `resource_group_name` - (Required) The name of the resource group in which to

Reviewer: we could simplify this by making this:

```suggestion
* `resource_group_name` - (Required) The name of the resource group where the Traffic Manager Profile exists.
```

Author: updated

>     create the Traffic Manager endpoint.

Reviewer: and removing this:

```suggestion
```

Author: updated

> ~> **Note:** "Traffic Manager Profile" and "Traffic Manager Endpoint" must be under same resource group.

Reviewer: and removing this:

```suggestion
```

Author: Updated
